### ISSUE TITLE
I need accuracy score for the code.

### ISSUE THREAD

**Issue by Affanabbbas** [yinkalario/General-Purpose-Sound-Recognition-Demo#4]
Hi sir, I found your code quite useful, but I need to have accuracy score for it for comparison. Please do the needful at the earliest.

**Comment by andres-fr** [comment]
What do you mean with accuracy score? This is a demo.
The corresponding research paper, with reported accuracy and other relevant metrics can be found here; https://arxiv.org/abs/1912.10211
The model used can be found [here](https://github.com/yinkalario/General-Purpose-Sound-Recognition-Demo/blob/dc9e54db1fb6a8c2855e2dfc27121d7b608083f7/sed_demo/models.py#L86)

We found this model to be the most adequate for our speed+quality requirements. Feel free to change it for a different one that suits your needs. You can find code for the other PANNs models [here](https://github.com/qiuqiangkong/audioset_tagging_cnn/blob/master/pytorch/models.py), as well as instructions to download the pretrained weights in the README.

**Comment by andres-fr** [comment]
Closing due to inactivity
